Adding PG enum drop, rename, add value, rename value (#44898)
rails/rails: activerecord/CHANGELOG.md

*   Added PostgreSQL migration commands for enum rename, add value, and rename value.

>     `rename_enum` and `rename_enum_value` are reversible though due to Postgres
>     limitation `add_enum_value` is not reversible since you cannot delete enum
>     values at this time and instead would have to drop and recreate the enum
>     entirely.

Reviewer: ```suggestion
    `rename_enum` and `rename_enum_value` are reversible. Due to Postgres
    limitation, `add_enum_value` is not reversible since you cannot delete enum
    values. As an alternative you would should drop and recreate the enum entirely.
```

Author: I've made this change, no more run on sentence, thank you!

*   Added PostgreSQL migration commands for enum drop, rename, add value, and

>     rename value.

Reviewer 2: Would you add some usage for each commands here?

Author: This has been added ❤️
